src/main/kotlin/org/rust/lang/core/psi/RsStructFiller.kt

```diff
@@ -0,0 +1,91 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.lang.core.psi
+
+import org.rust.lang.core.psi.ext.RsFieldDecl
+import org.rust.lang.core.psi.ext.RsMod
+import org.rust.lang.core.resolve.KnownItems
+import org.rust.lang.core.types.type
+
+class RsStructFiller(val items: KnownItems, val mod: RsMod, val psiFactory: RsPsiFactory, val recursive: Boolean) {
```

Comment:
I don't like this 2 class that refers to each other recursive call to each other and containing of the same properties. Maybe single class is enough?

Reply:
OK, now I combined it